config/prod/bgrande-nextflow-service-account.yaml

```diff
@@ -1,13 +1,12 @@
 # Provision a service account for AWS API access keys
-template_path: "remote/service-account.yaml"
+template:
+  type: "http"
+  url: "https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.2.7/IAM/service-account.yaml"
 stack_name: "bgrande-nextflow-service-account"
 stack_tags:
   Department: "CompOnc"
   Project: "imCORE"
   OwnerEmail: "bruno.grande@sagebase.org"
+  CostCenter: "Mobile Toolbox Project Core / 112501"
```

Comment:
How did you determine this cost center?  I guessed a different one for this project, as shown in this mapping: https://github.com/Sage-Bionetworks/aws-cost-accounting/blob/main/project_to_cost_center.tsv

Reply:
just a guess.  @BrunoGrandePhD can you please verify the most appropriate value for this?

Reply:
will remove with a  follow on PR.

Reply:
removed in PR https://github.com/Sage-Bionetworks/sandbox-provisioner/pull/402